src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorToKfsVendorDetailConversionServiceImpl.java

```diff
@@ -321,9 +325,12 @@ private VendorAddress assignStateCodeOrProvinceBasedOnFipsCountryCode(VendorAddr
     }
     
     private String convertFipsUsStateNameToFipsUsStateCode(String fipsStateName) {
-        List<org.kuali.rice.location.api.state.State> allStatesForUsFipsCountryCode =  new ArrayList<org.kuali.rice.location.api.state.State>();
-        allStatesForUsFipsCountryCode = getStateService().findAllStatesInCountry(KFSConstants.COUNTRY_CODE_UNITED_STATES);
-        List<org.kuali.rice.location.api.state.State> matchingStates = allStatesForUsFipsCountryCode.stream()
+        Map<String, Object> fieldValues = new HashMap();
+        fieldValues.put("active", true);
+        fieldValues.put("countryCode", KFSConstants.COUNTRY_CODE_UNITED_STATES);
+
+        Collection<State>  allStatesForUsFipsCountryCode = businessObjectService.findMatching(State.class, fieldValues);
+        List<State> matchingStates = allStatesForUsFipsCountryCode.stream()
```

Comment:
Log message on line 343 should probably be updated as well since we are no longer using the Rice tables.

Reply:
good catch, will update